tests/integration/api/test_api.py

```diff
@@ -206,14 +207,17 @@ def test_timeboard(self):
 
         assert dog.Timeboard.delete(timeboard["dash"]["id"]) is None
 
-    def test_search(self):
+    def test_search(self, dog):
         results = dog.Infrastructure.search(q="")
         assert len(results["results"]["hosts"]) > 0
         assert len(results["results"]["metrics"]) > 0
 
-    def test_metrics(self):
-        now = datetime.datetime.now()
-        now_ts = int(time.mktime(now.timetuple()))
+    @pytest.mark.skip
```

Comment:
@DataDog/integrations-tools-and-libraries I am having issues with recording and replaying this test case 👀

Reply:
What kind of issues ?